gen3-cli/cot/__init__.py

```diff
@@ -1,6 +1,7 @@
 # Here we controll modules loading order
 # We can do it in command.__init__ but I guess this is a little bit more explicit
 import cot.command  # noqa
+import cot.command.generate
```

Comment:
no `noqa`?

Reply:
That's interesting, but linter doesn't complain about this line.